mlabs/src/Mlabs/Nft/Contract/Server.hs

```diff
@@ -71,8 +71,9 @@ startNft StartParams{..} = do
     oref : _ -> do
       let nftId   = toNftId oref sp'content
           val     = SM.nftValue nftId
-          lookups = monetaryPolicy $ SM.nftPolicy nftId
-          tx      = mustForgeValue val
+          lookups = mintingPolicy $ SM.nftPolicy nftId
+          tx      = mustMintValue val 
+                    <> mustSpendPubKeyOutput oref
```

Comment:
I checked old version of plutus, which was used before update, and `mustSpendPubKeyOutput` was present there, so I have no thoughts atm, why tests were passing before. I remember `mustSpendPubKeyOutput` from plutus lectures, so I decided to try it here, and it worked. Without it, NFT contract tests fail with error `ScriptFailure (EvaluationError ["UTXO not consumed"])`, which thrown by validator in `Mlabs.Nft.Contract.Forge`